web/public/mgmt/config.yml

```diff
@@ -4308,6 +4355,19 @@ collections:
                         widget: object
                         fields:
                           - { label: "Description", name: "description", widget: "markdown" }
+                  - label: Non Essential Questions
+                    name: nonEssentialQuestions
+                    collapsed: true
+                    widget: object
+                    fields:
+                      - label: Default
+                        name: "default"
+                        collapsed: false
+                        widget: object
+                        fields:
+                          - { label: "Header", name: "header", widget: "string" }
+                          - { label: "Yes Text", name: "yesText", widget: "string" }
```

Comment:
I'm not 100% certain this is possible but I think you can give this a default text of yes via the default: property which would probably be nice for content